packages/discovery-provider/src/tasks/entity_manager/entities/playlist.py

```diff
@@ -661,6 +661,16 @@ def populate_playlist_record_metadata(
                         parsed_release_date
                     )
         elif key == "is_private":
+            # if playlist is being published (changing from private to public),
+            # override release_date and is_scheduled_release.
+            if (
+                playlist_record.is_private
+                and not playlist_metadata["is_private"]
+                and action == Action.UPDATE
+            ):
+                playlist_record.is_scheduled_release = False
+                playlist_record.release_date = str(datetime.now())  # type: ignore
```

Comment:
why the type ignore?

Reply:
copied code - i think `release_date` is a `DateTime` but we're setting it to string. works but wrong type.